spring-kafka-docs/src/main/asciidoc/kafka.adoc

```diff
@@ -5384,6 +5386,20 @@ If you wish to use a different error handling strategy for record and batch list
 |No replacements - use `DefaultErrorHandler` and throw an exception other than `BatchListenerFailedException`.
 |===
 
+[[migrating-legacy-eh]]
+====== Migrating Custom Legacy Error Handler Implementations to `CommonErrorHandler`
+
+Refer to the javadocs in `CommonErrorHandler`.
+
+To replace an `ErrorHandler` or `ConsumerAwareErrorHandler` implementation, you should implement `handleRecord()` and leave `remainingRecords()` to return `false` (default).
```

Comment:
The handleRecord() method implementation is deprecated starting from 2.9 and I am using SpringBoot 3.0.2.
Any thoughts on getting around this or an alternative/replacement?

Reply:
See the javadocs

https://github.com/spring-projects/spring-kafka/blob/73bee667dbaabca11ceee22a8d70b09934932d5f/spring-kafka/src/main/java/org/springframework/kafka/listener/CommonErrorHandler.java#L96-L97

I will fix the docs to reflect that.